colcon_core/location.py

```diff
@@ -130,23 +134,33 @@ def create_log_path():
         # create latest symlink
```

Comment:
Nit: create latest symlink**s**

Reply:
Fixed in 584206d2ef316739e5f34d7279fc9efe1592f812.